fastlane/lib/fastlane/actions/zip.rb

```diff
@@ -40,6 +40,16 @@ def self.available_options
         ]
       end
 
+      def self.example_code
+        [
+          "zip",
+          "zip(
```

Comment:
Just guessing, but perhaps they're two different code usage examples? (one plain, and one with params)